### ISSUE TITLE
Unpublished Podspec

### ISSUE THREAD

**Issue by CTOverton** [bambuser/BambuserPlayerSDK-iOS#16]
I noticed when the podspec was created yesterday that it noted the Firebase dependencies at the specific version 10.18.0, I made a pr that was merged last night I believe to instead use the minimum version 10.18.0. 

- #15 

However, though this pr was merged I don't believe the updates have been published to CocoaPods. When I try to use the `BambuserPlayerSDK` pod in my project I get the conflict

```
RNFBFirestore (from `../node_modules/@react-native-firebase/firestore`) was resolved to 19.3.0, which depends on
      Firebase/Firestore (= 10.24.0)

BambuserPlayerSDK (~> 1.5.3) was resolved to 1.5.3, which depends on
        Firebase/Firestore (= 10.18.0)
```

Once the changes to BambuserPlayerSDK pod have been published this should resolve the issue. It may need a version bump to `1.5.4` as well.

**Comment by saeidbasirnia** [comment]
Hi @CTOverton, 
`podspec` is updated in [specs](https://github.com/CocoaPods/Specs/blob/master/Specs/4/f/7/BambuserPlayerSDK/1.5.3/BambuserPlayerSDK.podspec.json).
Please make sure you cleared Cocoapods cache and `Podfile.lock`,
```bash
rm -f Podfile.lock
pod cache clean --all
```
and update your `podfile`:
```ruby
pod 'BambuserPlayerSDK', :git => 'https://github.com/bambuser/BambuserPlayerSDK-iOS.git', :tag => '1.5.3'
```
and then use 
```bash
pod install --repo-update
```
It should resolve the issue. Please let me know if you still facing the same issue.


<img width="1099" alt="image" src="https://github.com/user-attachments/assets/ff651f7f-3774-4e61-b69e-3d2df9b9012c">

**Comment by CTOverton** [comment]
@saeidbasirnia this is a good workaround for now, in the future though if you want to install this pod from the cocoapods resolution with just 
```
pod 'BambuserPlayerSDK', '~> 1.5.3'
```

I believe the pod needs to be published to the trunk, and since version 1.5.3 was already there it may need a version bump to 1.5.4 since it currently is still resolving with the Firebase requirement of 10.18.0 instead of the minimum. This would explain why using the direct Github git link for the pod works but using just the name has the older version.

https://guides.cocoapods.org/making/making-a-cocoapod.html#release

**Comment by CTOverton** [comment]
Seems to be working now, so maybe it was something on my end or maybe your team made a new publish haha, thank you anyways @saeidbasirnia! And I appreciate your teams response to issues it really helps 🙏
